rapthor/lib/miscellaneous.py

```diff
@@ -349,6 +349,31 @@ def delete_directory(dirname):
             raise e
 
 
+def ra2hhmmss(deg):
+    """Convert RA coordinate (in degrees) to HH MM SS"""
+
+    from math import modf
+    if deg < 0:
```

Comment:
This will break if `deg < -360`. And the function will also break if `deg > 360`.

Reply:
Indeed, I refactored this part and improved the documentation